ckanext/depositar_theme/fanstatic/webassets.yml

```
@@ -29,3 +35,20 @@ field-extra-js:
 p5-js:
   output: depositar_theme/%(version)s_p5.js
   contents: vendor/p5.js/p5.js
+
+font-awesome-css:
+  output: depositar_theme/%(version)s_font-awesome.css
+  filters: cssrewrite
+  contents: font-awesome/css/font-awesome.css
+
+jquery-js:
+  ouput: depositar_theme/%(version)s_jquery.js
+  contents: vendor/bootstrap-4.js/jquery.min.js
+
+bootstrap-4-js:
+  output: depositar_theme/%(version)s_bootstrap-4.js
+  contents: 
+    - vendor/bootstrap-4.js/bootstrap.js
+  extra:
+    preload:
+      - depositar_theme/jquery-js
```

使用 CKAN 內建 jquery，並刪除 `ckanext/depositar_theme/fanstatic/vendor/bootstrap-4.js/jquery.min.js`

```suggestion
      - vendor/jquery
```